delta-api/src/main/java/io/cdap/delta/api/DeltaTargetContext.java

```diff
@@ -27,17 +27,19 @@
    * Record that a DML operation was processed. Metrics on the number of operations processed are stored in memory
    * and actually written out when {@link #commitOffset(Offset, long)} is called.
    *
+   * @param tableName name of the source table for which count to be incremented
    * @param op type of DML operation
    */
-  void incrementCount(DMLOperation op);
+  void incrementCount(String tableName, DMLOperation op);
```

Comment:
instead of adding an argument, I think it would be better to create a new object that has table name and op as fields. This would make the API more extensible, in case we need add anything in the future.

Reply:
Refactored code to add DMLOperation and DDLOperation as classes with table name in them. Moved existing operation type enums into member variables of DMLOperation and DDLOperation. 

This method now takes DMLOperation.